vars/longevityPipeline.groovy

```diff
@@ -315,6 +330,21 @@ def call(Map pipelineParams) {
                     }
                 }
             }
+            stage('Finish Argus Test Run') {
```

Comment:
we'll need this stage on all pipelines, not just this one...

artifacts / rolling upgrades / manager / operator. and on siren-test as well.